test/e2e/performanceprofile/testdata/ppc-expected-profiles/profile1.yaml

```diff
@@ -18,3 +18,5 @@ spec:
     highPowerConsumption: false
     realTime: true
     perPodpowerManagement: false
+  net:
+    userLevelNetworking: false
```

Comment:
why did we have to add this specifically ?  (same comment for the rest of the files having the same change)

Reply:
Because we have a slight change in the way of how the flags are being initialized,  we added a default value for some of the variables to make the tests happy.